### PR TITLE
Feature/add users table

### LINKED ISSUE #14: Table to list all users
## Description

blocked by #11 

Create a component `UsersTable` with only one column:
- last name
- first name
- first 3 roles and a special ellipse 
  
In UsersPage, load ten first users from backend and display it in table.

Here is the call we have to do: `api/classes/User?limit=10`
See leto-modelizer for call headers.

image

### PR DESCRIPTION
**Pull request by k-y0u**
Resolves #14

### CODE REVIEW

File: src/components/tables/UsersTable.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: 'email',
+  required: true,
+  label: t('UsersTable.text.emailColumn'),
+  align: 'left',
+  field: 'email',
+}]);
+const users = ref([]);
+
+onMounted(async () => {
+  await UsersService.find().then((data) => {
```
Comment (Zorin95670) [inline src/components/tables/UsersTable.vue:45]: put this in search function like libraries.

File: tests/e2e/features/Users.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  ################## List Users ##################
+  ## 101 Should display all users
+
+  Scenario: Roundtrip about Users
+    Given I visit the '/'
+    When  I click on '[data-cy="drawer_item_users"]'
+    Then  I expect current url is '/users'
+
+    ## 101 Should display all users
+    And I expect '[data-cy="users_table"] tbody tr' appear 1 time on screen
```
Comment (Zorin95670) [inline tests/e2e/features/Users.feature:12]: check line content too like other test

Comment (k-y0u) [reply tests/e2e/features/Users.feature:12]: done

File: src/components/tables/UsersTable.vue (hunk 1 above)
Comment (k-y0u) [reply src/components/tables/UsersTable.vue]: done

File: tests/unit/services/UserService.test.js [hunk 3]
```diff
@@ -36,4 +36,19 @@ describe('Test: UserService', () => {
       expect(user.firstname).toEqual('Pradeep');
     });
   });
+
+  describe('Test function: find', () => {
+    it('should return all user information', async () => {
+      const users = [{
```
Comment (justinehell) [inline tests/unit/services/UserService.test.js:42]: I'd say `should return all userS information` since below you defined `const users`

**Review by justinehell — CHANGES_REQUESTED** [review]

**Review by Zorin95670 — DISMISSED** [review]

**Review by aboubakar14 — APPROVED** [review]

File: tests/unit/services/UserService.test.js (hunk 3 above)
Comment (k-y0u) [reply tests/unit/services/UserService.test.js]: done

**Review by justinehell — APPROVED** [review]

**Review by aboubakar14 — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**